### PR TITLE
feat: fiber opentelemetry trace example

### PR DESCRIPTION
**Pull request by psmarcin**
### Context
OpenTelemetry is starting to be standard way to collect any metadata about running application. This example shows how to integrate it with fiber and how to use OpenTelemetry context later in nested layers of application (base on context). 

Keep in mind that OpenTelemetry is complex piece of software and depends on many parts (e.g. collectors). This is not a scope of this example. Here we gonna use Lightstep as backend but it works with any other. 

### Changes
* [x] New example with opentelemetry integration
* [x] Comments in `main.go` for easier understanding
* [x] Screenshots what you will get at the end.

### CODE REVIEW

**Comment by TudorHulban** [comment]
Out of topic: Is there a Jaeger (jaegertracing.io) example?

**Comment by psmarcin** [comment]
Don't have much experience with Jeager but as far as I understand docs (https://www.jaegertracing.io/docs/1.21/opentelemetry/) it looks like it's OpenTelemetry compatible, so it should work with this example, but I didn't check it.

**Comment by psmarcin** [comment]
Still waiting for approval. Recently I've updated `fiber-otel` version to latest open-telemetry.

File: fiber-opentelemetry/README.md
Comment (koddr) [inline fiber-opentelemetry/README.md:10]: This line and next: why do you use `1.` for all list point? Can you please make normal numerical list.

File: fiber-opentelemetry/main.go
Comment (koddr) [inline fiber-opentelemetry/main.go:11]: Please give more instructions for each config points for users.

File: fiber-opentelemetry/main.go
Comment (koddr) [inline fiber-opentelemetry/main.go:23]: Please give more instructions for each midware points for users.

File: fiber-opentelemetry/README.md
Comment (psmarcin) [reply fiber-opentelemetry/README.md:10]: It's easier to move points if it's always `1.` at the beginning, what's more it doesn't matter for markdown. More here: https://www.markdownguide.org/basic-syntax/#ordered-lists

File: fiber-opentelemetry/main.go
Comment (psmarcin) [reply fiber-opentelemetry/main.go:12]: I think setting up open-telemetry connections and configuration is not a scope of `fiber-opentelemetry`, so I don't think it needs to be documented at all.

**Review by hi019 — APPROVED** [review]

**Review by koddr — APPROVED** [review]
LGTM

**Comment by psmarcin** [comment]
Any update?

**Comment by ReneWerner87** [comment]
thanks for the contribution

**Pull request merged**
